### ISSUE TITLE
NTR: Modifier: Shoulder predominance (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7841]
New term request Modifier: Shoulder predominance (elthat_150203172818):   UMLS: 0037004

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000163
- label: Shoulder localization
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated in (or predominant in) the shoulder.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7841
- synonyms:
